inc/vcf/file_structure.hpp

```diff
@@ -85,7 +85,31 @@ namespace ebi
             // TODO : SB metadata Type and Number is .
             { "SOMATIC", { "Flag", "0" } },
             { "VALIDATED", { "Flag", "0" } },
-            { "1000G", { "Flag", "0" } }
+            { "1000G", { "Flag", "0" } },
+            // tags specific to SVs are listed below
```

Comment:
Please sort these alphabetically to facilitate searching. Applies to the other maps too.